examples/c/ex_backup.c

```diff
@@ -205,126 +187,117 @@ take_full_backup(WT_SESSION *session, int i)
 				(void)snprintf(buf, sizeof(buf),
 				    "cp %s/%s %s/%s",
 				    home, filename, h, filename);
-				ret = system(buf);
+				error_check(system(buf));
 			}
 		else {
 			(void)snprintf(h, sizeof(h), "%s.%d", home_full, i);
 			(void)snprintf(buf, sizeof(buf), "cp %s/%s %s/%s",
 			    home, filename, hdir, filename);
-			ret = system(buf);
+			error_check(system(buf));
 		}
 	}
-	if (ret != WT_NOTFOUND)
-		fprintf(stderr,
-		    "WT_CURSOR.next: %s\n", session->strerror(session, ret));
-	ret = cursor->close(cursor);
-	return (ret);
+	scan_end_check(ret == WT_NOTFOUND);
+	error_check(cursor->close(cursor));
 }
 
-static int
+static void
 take_incr_backup(WT_SESSION *session, int i)
 {
 	WT_CURSOR *cursor;
 	int j, ret;
 	char buf[1024], h[256];
 	const char *filename;
 
-	ret = session->open_cursor(session, "backup:",
-	    NULL, "target=(\"log:\")", &cursor);
+	error_check(session->open_cursor(
+	    session, "backup:", NULL, "target=(\"log:\")", &cursor));
 
 	while ((ret = cursor->next(cursor)) == 0) {
-		ret = cursor->get_key(cursor, &filename);
+		error_check(cursor->get_key(cursor, &filename));
 		/*
 		 * Copy into the 0 incremental directory and then each of the
 		 * incremental directories for this iteration and later.
 		 */
 		(void)snprintf(h, sizeof(h), "%s.0", home_incr);
 		(void)snprintf(buf, sizeof(buf), "cp %s/%s %s/%s",
 		    home, filename, h, filename);
-		ret = system(buf);
+		error_check(system(buf));
 		for (j = i; j < MAX_ITERATIONS; j++) {
 			(void)snprintf(h, sizeof(h), "%s.%d", home_incr, j);
 			(void)snprintf(buf, sizeof(buf), "cp %s/%s %s/%s",
 			    home, filename, h, filename);
-			ret = system(buf);
+			error_check(system(buf));
 		}
 	}
-	if (ret != WT_NOTFOUND)
-		fprintf(stderr,
-		    "WT_CURSOR.next: %s\n", session->strerror(session, ret));
-	ret = 0;
+	scan_end_check(ret == WT_NOTFOUND);
+
 	/*
 	 * With an incremental cursor, we want to truncate on the backup
 	 * cursor to archive the logs.  Only do this if the copy process
 	 * was entirely successful.
 	 */
-	ret = session->truncate(session, "log:", cursor, NULL, NULL);
-	ret = cursor->close(cursor);
-	return (ret);
+	error_check(session->truncate(session, "log:", cursor, NULL, NULL));
+	error_check(cursor->close(cursor));
 }
 
 int
-main(void)
+main(int argc, char *argv[])
 {
 	WT_CONNECTION *wt_conn;
 	WT_SESSION *session;
-	int i, ret;
+	int i;
 	char cmd_buf[256];
 
+	(void)argc;					/* Unused variable */
+	(void)testutil_set_progname(argv);
+
 	(void)snprintf(cmd_buf, sizeof(cmd_buf),
 	    "rm -rf %s && mkdir %s", home, home);
```

Comment:
I think so, but I'll leave that up to @sueloverso: `ex_backup.c` has its own set of 3 databases, and the naming is clearly intended to align between the 3, so I was hesitant to change it.